### ISSUE TITLE
Suggestion: Forgive AFKs that happened because of bot issues

### ISSUE THREAD

**Issue by RainRat** [parabola949/Werewolf#97]
When a player is killed for AFK, bot reports the number of AFKs in the past 24 hours, but includes AFKs that are understandable because of known bot malfunctions. It's not feasible to go back through chat history to see if AFK is forgivable for each AFK report an admin receives. Admins should have a function to forgive (subtract from AFK totals) all AFKs that happened during a specific time frame. (Even better: bot recognizes it had trouble during specific timeframe and automatically forgives.)

**Comment by CloneMMDDCVII** [comment]
@RainRat How about the following : if four people at least die of afk a night, their counter will not be increased ?

**Comment by dllz** [comment]
@CloneMMDDCVII Looks like this would be a better solution as the afk was implented in order for us to detect people afking across the community as they are ruining the game.

**Comment by CloneMMDDCVII** [comment]
@dllz Yup :D true enough. We'll see with para when he comes back (i realise i now need to change my username on git)

**Comment by parabola949** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[Werewolf] Forgive AFKs that happen when bot status is not "Normal" (bot lags)](https://trello.com/c/tEyUaYXZ/8-werewolf-forgive-afks-that-happen-when-bot-status-is-not-normal-bot-lags)

**Comment by Olfi01** [comment]
This is not a bad idea, but in my opinion the importance is too low for the amount of effort this would take to realize, since it counts the amount of deaths by idling in the db directly and doesn't have any additional counter
